### PR TITLE
add endpoint to webpubsub swaggers

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

**Review by msyyc — APPROVED** [review]

File: specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json [hunk 1]
```diff
@@ -907,5 +916,16 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "Connection string or HTTP or HTTPS endpoint for the WebPubSub service instance.",
```
Comment (vicancy) [inline specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json:923]: Sounds a little confusing to me that this Endpoint can be also a connection string

Comment (vicancy) [reply specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json:923]: Endpoint sounds enough? Connection string needs parser or convenience layer after all

**Comment by vicancy** [comment]
Add @Mohit-Chakraborty @bterlson and @srnagar for awareness

File: specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json (hunk 1 above)
Comment (vicancy) [reply specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json:923]:
```suggestion
      "description": "HTTP or HTTPS endpoint for the WebPubSub service instance.",
```

**Review by vicancy — APPROVED** [review]

File: specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json (hunk 1 above)
Comment (iscai-msft) [reply specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json:923]: good point @vicancy , updating!

File: specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json [hunk 2]
```diff
@@ -907,5 +916,16 @@
         }
       }
     }
+  },
+  "parameters": {
+    "Endpoint": {
+      "name": "Endpoint",
+      "description": "HTTP or HTTPS endpoint for the WebPubSub service instance.",
```
Comment (Mohit-Chakraborty) [inline specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json:923]: I don't see the term 'WebPubSub' used in the documentation.
Should it be 'Web PubSub'? Or we can drop the word altogether and say 'for the service instance'.

**Review by Mohit-Chakraborty — APPROVED** [review]
Left one small comment.

File: specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json (hunk 2 above)
Comment (lmazuel) [reply specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json]: I just made the change

**Pull request merged**